app-crypt/keybase/keybase-1.0.47.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils systemd user versionator
```

Comment:
lets see, why do you need versionator at all? we have:

`https://github.com/keybase/client/archive/v1.0.48.tar.gz`

that said:

`SRC_URI="https://github.com/keybase/client/archive/v${PV}.tar.gz -> ${P}.tar.gz"`

Reply:
So this was probably wrong in the previous version, then :)
I'll try to clean it up.